src/olympia/abuse/tests/test_models.py

```diff
@@ -3136,6 +3118,27 @@ def test_notify_reviewer_decision_rejection_addon_already_disabled(self):
             not in mail.outbox[0].body
         )
 
+    def test_notify_reviewer_decision_legal_forward(self):
+        addon_developer = user_factory()
+        addon = addon_factory(users=[addon_developer], status=amo.STATUS_DISABLED)
+        decision = ContentDecision(addon=addon)
+        assert not CinderJob.objects.exists()
+        responses.add(
+            responses.POST,
+            f'{settings.CINDER_SERVER_URL}create_report',
+            json={'job_id': '123456'},
+            status=201,
+        )
+        self._test_notify_reviewer_decision(
+            decision,
+            amo.LOG.REQUEST_LEGAL,
+            None,
+            expect_create_decision_call=False,
+            expect_create_job_decision_call=False,
+            expect_email=False,
+        )
+        assert CinderJob.objects.get().job_id == '123456'
```

Comment:
TIL you can just call get() is that different from first()? Also, probably better to asset that CinderJob with job_id = `<your id>` exists or is this django magic for the same thing?

Reply:
`.get()` only succeeds when there is a single match so it's giving us an extra validation there aren't 0, or more than 1 CinderJobs.

Reply:
Nice!